corehq/apps/reports/util.py

```diff
@@ -344,9 +344,15 @@ def numcell(text, value=None, convert='int', raw=None):
 
 
 def datespan_from_beginning(domain_object, timezone):
-    startdate = domain_object.date_created
-    now = datetime.utcnow()
-    datespan = DateSpan(startdate, now, timezone=timezone)
+    # Start and end dates must be naive (no timezone) to work with DateSpan
+    startdate = domain_object.date_created  # full time in UTC
+    localized_start = startdate.astimezone(timezone)
```

Comment:
Just noting that this works because Django sets the system time to UTC so 'startdate' is assumed to be in UTC.

If you were to do this outside of Django and on a machine with system time set to something other than UTC we would need to first make `startdate` TZ aware: `pytz.utc.localize(value)`.

I think it's probably worth doing that since it is how we handle it in other places: `localized = ServerTime(date).user_time(timezone).done()`